lib/kast.js

```diff
@@ -322,12 +372,12 @@ const format = (o, isRaw = false, mixFix = false, colorize = false) => {
       , "_^Word__EVM-DATA"   : " ^W "
       , "_+Map__RULES"       : " +M "
       , "_andBool_"          : " AND "
-      , "_orBool__BOOL"      : " OR " 
-      , "_:__EVM-DATA"       : " : "  
+      , "_orBool__BOOL"      : " OR "
```

Comment:
Does K really have different labels for OR and AND? (meaning, the additional `_BOOL` here)

Reply:
yes, i think and is deeply buildin and or is provided by a library ontop of and